english/wake-up/wake-up-again/dream2/dream2.md

```diff
@@ -3,5 +3,10 @@ You look around and you see big white cyclinders everywhere on all four sides.
 Suddenly an aircraft flies by. 
 In the shiny underbelly of the airplane you see a sea of white cyclinders stretched across the landscape. 
 You realize that you are just a marshmallow among thousands in a vast island of marshmallows.
-Another marshmallow takes you to the marshmallow chief, who explains to you that
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.